providers/SolanaAccountsProvider.tsx

```diff
@@ -182,6 +183,74 @@ export const deserializeAccountInfos = (
                   ...(accountInfo as AccountInfo<Buffer>),
                   parsed: accountData.parsed?.info as ParsedTokenAccountData,
                 }
+          // taken from account deserialization in splToken getMintInfo
+        } else if (
```

Comment:
cc @gchatz22 for 👀 had to add this absolutely heinous account deserialization code to handle the non "parsed" encoding of getMultipleAccounts that no longer is allowed in newer web3.js - this is some code i was trying to avoid adding for awhile until we just went to new token client

this issue has plagued us for so long, next PR ill just upgrade it to the newer spl-token client, but I believe well have to update everywhere we read from mint / tokenaccounts to use that new type since its slightly different so covering that in separate PR

Thanks for the add @jordansexton 👍